app/build.gradle

```diff
@@ -31,6 +31,12 @@ android {
         sourceCompatibility JavaVersion.VERSION_1_8
         targetCompatibility JavaVersion.VERSION_1_8
     }
+    // Coil requires Java 8 bytecode.
+    // Enables Java 8 for this Kotlin project.
+    kotlinOptions {
+        jvmTarget = "1.8"
+    }
+
```

Comment:
I think it would make sense to add this to the project `build.gradle` so all the modules get the same configuration:
```
tasks.withType(KotlinCompile).all {
        kotlinOptions {
            jvmTarget = "1.8"
        }
    }
```

Reply:
Agreed. I'll enable the above option in the project level `build.gradle` .

Reply:
Awesome. Let me know when you update the PR so we can merge it 🎉

Reply:
Hey! I have updated the PR. Please review.